### PR TITLE
rewrite boc topic

### PR DESCRIPTION
**Pull request by cryshado**

### CODE REVIEW

**Review by ilyar — APPROVED** [review]

File: src/arch/07-tlb-and-boc.md [hunk 1]
```diff
@@ -142,18 +142,12 @@ It is done recursively turning to the serialization of variables (they are TL-B
...
-
-If you have accounts in your ERC-20 token that contain pennies, then the fees for holding these accounts will greatly exceed the value of these accounts over the years. Therefore, in ES it is customary to make separate contracts for separate accounts, which themselves pay for their storage. 
-
-Fun fact: this entire overcomplicated data storage scheme helps to scale the network, so that the validator can quickly execute smart contracts. Validators do not need to store the entire current state of the blockchain in Ram, because loading the smart contract state from the disk will be fast, since all smart contracts are small. Additionally, since all contracts are small, they are evenly distributed across all shards.
+Read more about bag of cells in the [tvm.pdf](https://ton.org/tvm.pdf) and [tblkch.pdf](https://ton.org/tblkch.pdf) documents.
```
Comment (ilyar) [inline src/arch/07-tlb-and-boc.md:153]: https://docs.everscale.network/tvm.pdf

**Review by ilyar — CHANGES_REQUESTED** [review]

**Review by ilyar — CHANGES_REQUESTED** [review]
image
Please fix your commit signature

**Comment by ilyar** [comment]
@everscale-org/docs **preview**  https://everscale-org.github.io/preview/PR-362/arch/tlb-and-boc#boc

File: src/arch/07-tlb-and-boc.md (hunk 1 above)
Comment (cryshado) [reply src/arch/07-tlb-and-boc.md:153]: > https://docs.everscale.network/tvm.pdf

The link you provided contains a non-original version. I'm not sure if it is correct

**Review by nerzh — APPROVED** [review]

**Review by ch1seL — APPROVED** [review]

**Review by ilyar — APPROVED** [review]

**Pull request merged**